pkg/services/export/export_auth.go

```diff
@@ -13,58 +15,116 @@ func dumpAuthTables(helper *commitHelper, job *gitExportJob) error {
-					Cols("permission.*").
-					Where("org_id = ?", helper.orgID).
-					Asc("permission.id")
-			case "user":
-				sess.Table(table).
-					Join("inner", "org_user", "user.id = org_user.user_id").
-					Cols("user.*", "org_user.role").
-					Where("org_user.org_id = ?", helper.orgID).
-					Asc("user.id")
-			case "user_auth_token":
-				sess.Table(table).
-					Join("inner", "org_user", "user_auth_token.id = org_user.user_id").
-					Cols("user_auth_token.*").
-					Where("org_user.org_id = ?", helper.orgID).
-					Asc("user_auth_token.id")
-			default:
-				sess.Table(table).Where("org_id = ?", helper.orgID).Asc("id")
+		dump := []statsTables{
+			{
+				table: "user",
+				sql: `
+					SELECT user.*, org_user.role 
+					  FROM user 
+					  JOIN org_user ON user.id = org_user.user_id
+					 WHERE org_user.org_id =` + strconv.FormatInt(helper.orgID, 10),
+				converters: []sqlutil.Converter{{Dynamic: true}},
+				drop: []string{
+					"id", "version",
+					"password", // UMMMMM... for now
+					"org_id",
+				},
+			},
+			{
+				table: "user_role",
+				sql: `
+					SELECT * FROM user_role 
+					 WHERE org_id =` + strconv.FormatInt(helper.orgID, 10),
+			},
+			{
+				table: "builtin_role",
+				sql: `
+					SELECT * FROM builtin_role 
+					 WHERE org_id =` + strconv.FormatInt(helper.orgID, 10),
+			},
+			{
+				table: "api_key",
+				sql: `
+					SELECT * FROM api_key 
+					 WHERE org_id =` + strconv.FormatInt(helper.orgID, 10),
+			},
+			{
+				table: "permission",
+				sql: `
+					SELECT permission.* 
+					  FROM permission 
+					  JOIN role ON permission.role_id = role.id
+					 WHERE org_id =` + strconv.FormatInt(helper.orgID, 10),
+			},
+			{
+				table: "user_auth_token",
+				sql: `
+					SELECT user_auth_token.* 
+					  FROM user_auth_token 
+					  JOIN org_user ON user_auth_token.id = org_user.user_id
+					 WHERE org_user.org_id =` + strconv.FormatInt(helper.orgID, 10),
+			},
+			{table: "team"},
+			{table: "team_group"},
+			{table: "team_role"},
+			{table: "team_member"},
+			{table: "temp_user"},
+			{table: "role"},
+		}
+
+		for _, auth := range dump {
+			if auth.sql == "" {
+				auth.sql = `
+					SELECT * FROM ` + auth.table + ` 
+					 WHERE org_id =` + strconv.FormatInt(helper.orgID, 10)
+			}
+			if auth.converters == nil {
+				auth.converters = []sqlutil.Converter{{Dynamic: true}}
+			}
+			if auth.drop == nil {
+				auth.drop = []string{
+					"id",
+					"org_id",
+				}
 			}
 
-			raw, err := sess.QueryInterface()
+			rows, err := sess.DB().QueryContext(helper.ctx, auth.sql)
 			if err != nil {
-				return fmt.Errorf("unable to read: %s // %s", table, err.Error())
-			}
-			if len(raw) < 1 {
-				continue // don't write empty files
+				return err
 			}
-			frame, err := queryResultToDataFrame(raw, frameOpts{
-				skip: []string{"org_id", "version", "help_flags1", "theme"},
-			})
+
+			frame, err := sqlutil.FrameFromRows(rows.Rows, -1, auth.converters...)
```

Comment:
@ArturWierzbicki -- in a previous PR, you asked for tests on how the `queryResultToDataFrame` works.  In this PR, I am removing the whole `frame_helper.go` and using the version from our SDK https://github.com/grafana/grafana-plugin-sdk-go/blob/main/data/sqlutil/sql.go#L22